pkgs/development/python-modules/environs/default.nix

```diff
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "environs";
-  version = "9.3.4";
+  version = "9.3.5";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
```

Comment:
I'm not sure this is needed, because the oldest Python3 version in nixpkgs is 3.7.